core/psd/imx6ull/psd.c

```diff
@@ -405,6 +407,18 @@ static int psd_writeFile(sdp_cmd_t *cmd)
 	return err;
 }
 
+static void *wdg_kicker_thr(void *args)
+{
+	unsigned timeout = (unsigned)args;
+
+	while (1) {
+		wdgreload();
+		usleep(timeout * 1000L);
```

Comment:
side portability note: `usleep` might not support arguments > 1000000 (we do support them so not a merge blocker)